src/fastmcp/server/auth/providers/discord.py

```
@@ -0,0 +1,322 @@
+                    "allowed_client_redirect_uris": allowed_client_redirect_uris,
+                    "jwt_signing_key": jwt_signing_key,
+                }.items()
+                if v is not NotSet
+            }
+        )
+
+        # Validate required settings
+        if not settings.client_id:
+            raise ValueError(
+                "client_id is required - set via parameter or FASTMCP_SERVER_AUTH_DISCORD_CLIENT_ID"
+            )
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)